static/css/app.css

```diff
@@ -334,6 +334,10 @@ section,
 summary {
     display: block;
     padding-bottom: 0.5%;
+    padding-inline-start: 1.5%;
```

Comment:
If you just want to fix the footer, why did you put it in here? Now this will apply to all those elements as well. That makes no sense.

Reply:
I can use a separate id or class for this but the changes now doesn't affect others because of their orientation.

Reply:
Still, I'd like to see it in a separate section.

Reply:
@canihavesomecoffee but it's better this way as it aligned with symmetry with others